lib/mcinstall/index.js

```diff
@@ -0,0 +1,64 @@
+import {BaseServicePlist} from '../base-service';
+import {fs, plist} from '@appium/support';
+
+
+const MC_INSTALL_SERVICE_NAME = 'com.apple.mobile.MCInstall';
+const ACKNOWLEDGED = 'Acknowledged';
+
+class MCInstallProxyService extends BaseServicePlist {
+  constructor (socketClient) {
+    super(socketClient);
+  }
+
+  checkACK (res, name) {
+    if (res.Status !== ACKNOWLEDGED) {
+      throw new Error(`${name} error: ${JSON.stringify(res)}`);
+    }
```

Comment:
we could return `res` from here, so the resulting invocation would be shorter (e.g.

```
     this.checkACK(res, 'getProfileList');
     return res;
```
becomes
```
    return checkAck(res, 'getProfileList');
```
)